pilot/cmd/pilot-agent/main.go

```diff
@@ -327,6 +335,12 @@ var (
 					if disableInternalTelemetry {
 						opts["DisableReportCalls"] = "true"
 					}
+
```

Comment:
This code path is not used in the new installer ( see the TODO above on deprecating the custom bootstrap parser)